packages/network/src/logic/tracker/InstructionCounter.ts

```diff
@@ -1,43 +1,45 @@
-import { Status, StreamKey } from '../../identifiers'
+import { SPID, SPIDKey } from 'streamr-client-protocol'
+import { Status } from '../../identifiers'
 import { NodeId } from '../node/Node'
 
 export const COUNTER_UNSUBSCRIBE = -1
 
-type Counters = Record<NodeId,Record<StreamKey,number>>
+type Counters = Record<NodeId,Record<SPIDKey,number>>
```

Comment:
I find it hard to parse these without spaces tbh:
```suggestion
type Counters = Record<NodeId, Record<SPIDKey, number>>
```